File: Darts/src/burlton/dartzee/code/screen/game/MatchStatisticsPanelRoundTheClock.java
package burlton.dartzee.code.screen.game;

import java.util.ArrayList;

public class MatchStatisticsPanelRoundTheClock extends GameStatisticsPanelRoundTheClock
{
	@Override
	protected void addRowsToTable()
	{
		super.addRowsToTable();
		
		addRow(new Object[getRowWidth()]);
		addRow(getBestGameRow(is -> is.min()));
		addRow(getAverageGameRow());
	}
	
	@Override
	protected ArrayList<Integer> getRankedRowsLowestWins()
	{
		ArrayList<Integer> ret = super.getRankedRowsLowestWins();
		ret.add(16);
		ret.add(17);
		return ret;
	}
}

File: Darts/src/burlton/dartzee/code/screen/game/MatchStatisticsPanelRoundTheClock.kt
package burlton.dartzee.code.screen.game

class MatchStatisticsPanelRoundTheClock : GameStatisticsPanelRoundTheClock()
{
    override fun addRowsToTable()
    {
        super.addRowsToTable()

        addRow(arrayOfNulls(getRowWidth()))
        addRow(getBestGameRow { stream -> stream.min() })
        addRow(getAverageGameRow())
    }

    override fun getRankedRowsLowestWins(): MutableList<String>
    {
        val ret = super.getRankedRowsLowestWins()
        ret.add("Best Game")
        ret.add("Avg Game")
        return ret
    }
}
